nginx/default.toml

```diff
@@ -34,5 +40,18 @@ tcp_nodelay = "on"
 # http.keepalive_timeout: Timeout on client connection keepalive, in seconds. Default = 75
 keepalive_timeout = 60
 
+# http.server_names_hash_bucket_size: Sets the bucket size for the server names hash tables.
+# The default value depends on the size of the processor’s cache line. Default = 32
+server_names_hash_bucket_size = 64
+
+www-data = "/hab/svc/nginx/data"
+global_ssl_password_file = "# ssl_password_file /home/hab/.ssh/tls/global.pass;"
```

Comment:
So we saw this pattern of specifying any top-level directives, but this quickly becomes hard to maintain, since anyone who wants to add a custom directive would then need to add:
- new directive to the template
- configuration to set the value of the directive.

A pattern that's proven much more flexible is some manner of, ERB example:

```erb
<% @directives.sort_by { |key, val| key }.each do |directive, value| -%>
<%= "#{directive} #{value};" %>
<% end -%>
```

This leaves the implementation up to the author to add any directives they'd like.